src/unittest/python/test_api.py

```diff
@@ -1,6 +1,7 @@
 
 import unittest
-from src.api import Api
+import os
```

Comment:
Minor nit: It looks like we're just using the `environ` method of `os` so I would say it's probably better to just import that function instead of the whole module.

Reply:
good call, runtime gains are gains no matter how small